crates/anvil/tests/it/optimism.rs

```diff
@@ -8,19 +8,15 @@ use alloy_provider::Provider;
 use alloy_rpc_types::{optimism::OptimismTransactionFields, TransactionRequest, WithOtherFields};
 use anvil::{spawn, Hardfork, NodeConfig};
 
-// TODO: transaction is expected to fail, it does not, remove ignore once fixed
 #[tokio::test(flavor = "multi_thread")]
-#[ignore]
```

Comment:
`send` / `send_transaction` fail with gas estimation layer because the node returns an RPC error if the call/estimation fails, whereas without the layer the transaction will be sent directly to the node, returning a receipt with `"status": false`.